frontend/src/js/controllers/challengeHostTeamsCtrl.js

```diff
@@ -138,6 +137,86 @@
 
         utilities.sendRequest(parameters);
 
+
+        vm.showMdDialog = function(ev, hostTeamId) {
+            vm.hostTeamId = hostTeamId;
+            parameters.url = 'hosts/challenge_host_team/' + hostTeamId;
+            parameters.method = 'GET';
+            parameters.token = userKey;
+            parameters.callback = {
+                onSuccess: function(response) {
+                    vm.team.name_modal = response.data.team_name;
+                    vm.team.url_modal = response.data.team_url;
+                },
+                onError: function(response) {
+                    var error = response.data['error'];
+                    vm.stopLoader();
+                    $rootScope.notify("error", error);
+                }
+            };
+            utilities.sendRequest(parameters);
+
+            $mdDialog.show({
+                scope: $scope,
+                preserveScope: true,
+                targetEvent: ev,
+                templateUrl: 'dist/views/web/edit-host-teams.html'
+            });
+        };
+
+
+        vm.updateHostTeamData = function(updateHostTeamDataForm) {
+            if (updateHostTeamDataForm) {
+            var parameters = {};
+            parameters.url = 'hosts/challenge_host_team/' + vm.hostTeamId;
+            parameters.method = 'PATCH';
+            parameters.data = {
+                "team_name": vm.team.name_modal,
+                "team_url": vm.team.url_modal
```

Comment:
Please modify it as per above variable names.

Reply:
done.